ui/src/store/store.ts

```diff
@@ -0,0 +1,15 @@
+// Package import
+import { configureStore } from '@reduxjs/toolkit';
+
+// rtk queries import
+import { api } from './api';
+
+const store = configureStore({
+  reducer: {
+    [api.reducerPath]: api.reducer,
+  },
+  middleware: (getDefaultMiddleware) =>
+    getDefaultMiddleware().concat(api.middleware),
```

Comment:
what are default middlewares?

Reply:
There are some prefined middlewares like thunk in redux toolkit. We are adding api.middleware in the predefined middlewares